chia/full_node/full_node.py

```diff
@@ -291,7 +291,10 @@ def get_connections(self, request_node_type: Optional[NodeType]) -> List[Dict[st
             peak_store = None
         for con in connections:
             if peak_store is not None and con.peer_node_id in peak_store:
-                peak_hash, peak_height, peak_weight = peak_store[con.peer_node_id]
+                peak = peak_store[con.peer_node_id]
+                peak_height = peak.height
+                peak_hash = peak.header_hash
```

Comment:
im not sure where and how get_connections is used
but i think it would be nicer to just use Peak here and not separate to each field

Reply:
It's done like this to retain the current behaviour because it's used by an RPC endpoint.

Reply:
yes i know just saying would be nicer to use your new class, not insisting on this as im not sure how much code depends on this

Reply:
Agree and i thought about it too but i think we shouldn't just break compatibility. At the end it could have been done with a `peak` object before this PR too without the `Peak` class.